main.rb

```diff
@@ -0,0 +1,37 @@
+require_relative 'app'
+
+app = App.new
+
+puts "Welcome to the School Library! \n\n"
+
+loop do
+  puts 'Choose an option:'
+  puts '1. List all books'
+  puts '2. List all people'
+  puts '3. Create a person'
+  puts '4. Create a book'
+  puts '5. Create a rental'
+  puts '6. List all rentals for a given person ID'
+  puts '7. Quit'
+  option = gets.chomp
+
+  case option
+  when '1'
+    app.list_books
+  when '2'
+    app.list_persons
+  when '3'
+    app.create_person
+  when '4'
+    app.create_book
+  when '5'
+    app.create_rental
+  when '6'
+    app.list_rentals_by_person_id
+  when '7'
+    print 'Thank you for using this app!'
+    break
```

Comment:
- [ ] Since I could not create a person there was no point checking for these other options as they require the creation of a person. Kindly ensure these options also work as required.

  Kindly look at the video that was shared with you and make your app work just like it.

Reply:
- I was able to create a book and list all books but I was unable to create Rental nor list them because of the above error.